app/scripts/modules/openstack/serverGroup/configure/wizard/serverGroupWizard.html

```diff
@@ -0,0 +1,31 @@
+<div>
+  <div ng-if="state.requiresTemplateSelection">
+    <ng-include src="pages.templateSelection"></ng-include>
+  </div>
+  <div ng-if="!state.loaded" style="height: 200px">
+    <h3 us-spinner="{radius:30, width:8, length: 16}"></h3>
+  </div>
+  <div>
+    <form name="serverGroupWizardForm" class="form-horizontal" novalidate>
+      <v2-modal-wizard ng-show="state.loaded && !state.requiresTemplateSelection" heading="{{title}}">
+        <v2-wizard-page key="basic-settings" label="Basic Settings" hide-subheading="true">
+          <ng-include src="pages.basicSettings"></ng-include>
+        </v2-wizard-page>
+        <v2-wizard-page key="load-balancers" label="Load Balancers" mark-complete-on-view="false">
+          <ng-include src="pages.loadBalancers"></ng-include>
+        </v2-wizard-page>
+        <v2-wizard-page key="security-groups" label="Security Groups" mark-complete-on-view="false" done="true">
+          <ng-include src="pages.securityGroups"></ng-include>
+        </v2-wizard-page>
```

Comment:
Security groups will be needed in Openstack, so I'd like to leave this here to be implemented in another PR